docs/cli-reference/dfx-canister.md

```diff
@@ -555,7 +555,7 @@ Use the `dfx canister request-status` command to request the status of a specifi
 ### Basic usage
 
 ``` bash
-dfx canister request-status request_id [option]
+dfx canister request-status [OPTIONS] <REQUEST_ID> <CANISTER>
```

Comment:
Please follow the same patterns used in all of the other command documentation here.
```suggestion
dfx canister request-status request_id canister <option>
```